src/internal/NeoEsp8266UartMethod.h

```diff
@@ -95,43 +119,34 @@ template<typename T_SPEED> class NeoEsp8266UartMethodBase
         // subsequent round of data until the latch time has elapsed.  This
         // allows the mainline code to start generating the next frame of data
         // rather than stalling for the latch.
-        
-        while (!IsReadyToUpdate())
+        while (!this->IsReadyToUpdate())
         {
             yield();
         }
-        
-        // since uart is async buffer send, we have to calc the endtime that it will take
-        // to correctly manage the data latch in the above code
-        // add the calculated time to the current time 
-        _endTime = micros() + getPixelTime();
-
-        // esp hardware uart sending of data
-        esp8266_uart1_send_pixels(_pixels, _pixels + _sizePixels);
+        this->UpdateUart();
     }
 
     uint8_t* getPixels() const
     {
-        return _pixels;
+        return this->_pixels;
     };
 
     size_t getPixelsSize() const
     {
-        return _sizePixels;
+        return this->_sizePixels;
     };
 
 private:
     uint32_t getPixelTime() const
     {
-        return (T_SPEED::ByteSendTimeUs * _sizePixels);
+        return (T_SPEED::ByteSendTimeUs * this->_sizePixels);
```

Comment:
Not in this case. _sizePixel is defined in a base class that is a template parameter.